crates/epoch-snark/src/gadgets/epochs.rs

```diff
@@ -87,16 +87,16 @@ impl ConstraintSynthesizer<Fr> for ValidatorSetUpdate<Bls12_377> {
         let _enter = span.enter();
         info!("generating constraints");
         let epoch_bits = self.enforce(cs)?;
-        let x = epoch_bits.first_epoch_bits[0].cs();
-        epoch_bits.verify(self.hash_helper, x)?;
-
+        let cs = epoch_bits.first_epoch_bits[0].cs();
```

Comment:
Should iterator through all of the bits for a constraint system instead of indexing here at 0